packages/indicators/src/__tests__/__integration/indicatorApi.fixtures.json

```diff
@@ -41,15 +53,51 @@
     },
     "surveys": [
       {
-        "code": "Births",
+        "code": "FavouriteColour",
         "questions": [
-          { "code": "Female", "type": "Number" },
-          { "code": "Male", "type": "Number" }
+          { "code": "Red", "type": "Number" },
+          { "code": "Blue", "type": "Number" }
         ]
       }
+    ],
+    "surveyResponses": [
+      {
+        "surveyCode": "Births",
+        "entityCode": "AU",
+        "submission_time": "2020-12-10T15:00:00Z",
+        "answers": {
+          "Red": "10",
+          "Blue": "20"
```

Comment:
Red and Blue births sound a bit morbid... 😅

Reply:
Indeed! 😆 Fixed in https://github.com/beyondessential/tupaia/commit/0746213a7b99f994f34a8d670b698d53a84bd0f7 (generally a housekeeping commit, but this change sneaked in)